python/ray/rllib/env/env_context.py

```diff
@@ -20,13 +20,26 @@ class EnvContext(dict):
             uniquely identifies the worker the env is created in.
         vector_index (int): When there are multiple envs per worker, this
             uniquely identifies the env index within the worker.
+        remote (bool): Whether environment should be remote or not.
+        entangled_envs_num (int): If environment is entangled,
+            how much logical environments does it have
     """
 
-    def __init__(self, env_config, worker_index, vector_index=0):
+    def __init__(self, env_config, worker_index, vector_index=0, remote=False,
+                 entangled_envs_num=0):
         dict.__init__(self, env_config)
         self.worker_index = worker_index
         self.vector_index = vector_index
+        self.remote = remote
+        self.entangled_envs_num = entangled_envs_num
 
-    def with_vector_index(self, vector_index):
+    def align(self, env_config=None, worker_index=None, vector_index=None,
```

Comment:
What does this function do?

Reply:
Bad naming, agree. Renamed to copy_with_overrides.